### ISSUE TITLE
Typo

### ISSUE THREAD

**Issue by aliunlu97** [mml-book/mml-book.github.io#391]
**Describe the mistake**
It is stated that "We partition our training set into two sets \mathcal{D} =  \mathcal{R} \cup \mathcal{V}, such that they do not overlap (\mathcal{R} \cap \mathcal{V} = \emptyset), where \mathcal{V} is the validation set". 

Before this statement, it is also stated that "K-fold cross-validation effectively partitions the data into K chunks, K − 1 of which form the training set \mathcal{R}, and the last chunk serves as the validation set \mathcal{V}"

From these two statements, it can be inferred that the first one I hereby quoted should instead be "We partition our dataset into two sets \mathcal{D} = \mathcal{R} \cup \mathcal{V}" since both \mathcal{D} and \mathcal{R} cannot be training sets.

Sincerely,

**Location**
Please provide the 
1. 2019-08-15
2. 8.2.4
3. 263
4. 35

**Proposed solution**
Change training set to dataset in the sentence "We partition our training set into two sets \mathcal{D} =  \mathcal{R} \cup \mathcal{V} ..."
